### PR TITLE
Add Dark Theme 1.0

### PR DESCRIPTION
**Pull request by daniel8lopes**

### CODE REVIEW

File: Themes/DarkMode.scss [hunk 1]
```diff
@@ -0,0 +1,177 @@
+// OMNIA Low-Code
+// Dark Theme 1.0
+
+//
+// Color system
+//
+
+$white:    #fff !default;
```
Comment (gsferreira) [inline Themes/DarkMode.scss]: remove the !default tag from all variables.

File: Themes/DarkMode.scss [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+// Links
+
+$link-color:                $blue !default;
+
+// Fonts
+
+$font-family-sans-serif:      "Lato", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol" !default;
+
+//$font-size-base:              0.9375rem !default;
```
Comment (gsferreira) [inline Themes/DarkMode.scss]: If we don't need it, please remove.

File: Themes/DarkMode.scss (hunk 1 above)
Comment (daniel8lopes) [reply Themes/DarkMode.scss]: Done

File: Themes/DarkMode.scss (hunk 2 above)
Comment (daniel8lopes) [reply Themes/DarkMode.scss]: Done

**Review by zebilha — APPROVED** [review]

**Pull request merged**